### PR TITLE
Map some constants found in the world classes

### PR DESCRIPTION
**Pull request by Sollace**

### CODE REVIEW

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 1]
```diff
@@ -22,6 +22,8 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	FIELD field_26934 entityList Lnet/minecraft/class_5574;
 	FIELD field_26935 entityManager Lnet/minecraft/class_5579;
 	FIELD field_28859 sleepManager Lnet/minecraft/class_5838;
+	FIELD field_29768 ENTITY_TICK_FREQUENCY I
+		COMMENT The number of ticks the world will wait between entity updates.
```
Comment (liach) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: Hmm, so this is not `1`? Interesting, I thought entities are always ticked

File: mappings/net/minecraft/world/World.mapping [hunk 2]
```diff
@@ -19,7 +23,7 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 	FIELD field_9234 thunderGradient F
 	FIELD field_9235 rainGradient F
 	FIELD field_9236 isClient Z
-	FIELD field_9238 unusedIncrement I
+	FIELD field_9238 lcgBlockSeedIncrement I
```
Comment (liach) [inline mappings/net/minecraft/world/World.mapping:26]: Just curious, what is `lcg`?

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]:
```suggestion
		COMMENT The number of ticks ({@value}) the world will wait between entity updates.
```
Javadoc will render the value on the web page with this little tag.

File: mappings/net/minecraft/world/World.mapping (hunk 2 above)
Comment (enbrain) [reply mappings/net/minecraft/world/World.mapping:26]: It's linear congruential generator. Might be worth documenting it if this abbreviation is not so common.

Comment (Sollace) [reply mappings/net/minecraft/world/World.mapping:26]: The abbreviation was used in existing names. I just copied the convention. If you want it documented we can do that in a separate PR.

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 1 above)
Comment (Sollace) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: @liach I thought so too, but evidently not. At least, I think I should change the description a little. Looks like it only applies when the server is idle.

```
        boolean bl2 = bl4 = !this.players.isEmpty() || !this.getForcedChunks().isEmpty();
        if (bl4) {
            this.resetIdleTimeout();
        }
        if (bl4 || this.idleTimeout++ < 300 /* ENTITY_TICK_FREQUENCY */) {
            lv.push("entities");
```

**Review by liach — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**